packages/cli/src/commands/export/credentials.ts

```diff
@@ -110,7 +111,8 @@ export class ExportCredentialsCommand extends BaseCommand {
 			findQuery.id = flags.id;
 		}
 
-		const credentials: ICredentialsDb[] = await Db.collections.Credentials.findBy(findQuery);
+		const credentials: ICredentialsDb[] =
```

Comment:
We might not need the typing here.

Reply:
unfortunately we still do. `ICredentialsDb` and `CredentialsEntity` are still two different types.